[Bug] [API] import processDefinition report json parse error

### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/dolphinscheduler/issues?q=is%3Aissue) and found no similar issues.


### What happened

[ERROR] 2021-10-20 20:40:03.391 org.apache.dolphinscheduler.common.utils.JSONUtils:[167] - parse list exception!
com.fasterxml.jackson.databind.exc.InvalidDefinitionException: Cannot construct instance of `org.apache.dolphinscheduler.api.dto.DagDataSchedule` (no Creators, like default construct, exist): cannot deserialize from Object value (no delegate- or property-based Creator)
 at [Source: (String)"[{"processDefinition":{"id":73,"code":818174268579840,"name":"hello-shell-2","version":1,"releaseState":"OFFLINE","projectCode":724559144984576,"description":"","globalParams":"[]","globalParamList":[],"globalParamMap":{},"createTime":"2021-10-16 23:44:57","updateTime":"2021-10-16 23:44:57","flag":"YES","userId":1,"userName":null,"projectName":null,"locations":"[{\"taskCode\":818173332135936,\"x\":202,\"y\":376}]","scheduleReleaseState":null,"timeout":0,"tenantId":-1,"tenantCode":null,"modifyBy""[truncated 1139 chars]; line: 1, column: 3] (through reference chain: java.util.ArrayList[0])
        at com.fasterxml.jackson.databind.exc.InvalidDefinitionException.from(InvalidDefinitionException.java:67)
        at com.fasterxml.jackson.databind.DeserializationContext.reportBadDefinition(DeserializationContext.java:1592)

### What you expected to happen

It can be import successfully

### How to reproduce

Export a processDefinition and then import it

### Anything else

_No response_

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

--- a/dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/dto/DagDataSchedule.java
+++ b/dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/dto/DagDataSchedule.java
@@ -30,6 +30,9 @@ public class DagDataSchedule extends DagData {
      */
     private Schedule schedule;
 
+    public DagDataSchedule() {
+    }
+
     public DagDataSchedule(DagData dagData) {
         super();
         this.setProcessDefinition(dagData.getProcessDefinition());

